docs/source/11_tools_integration/01_pyspark.md

```diff
@@ -26,6 +26,7 @@ from typing import Any, Dict, Union
 
 from pyspark import SparkConf
 from pyspark.sql import SparkSession
+from kedro.framework.context import KedroContext
```

Comment:
```suggestion
from kedro.framework.context import KedroContext
from pyspark import SparkConf
from pyspark.sql import SparkSession
```
Nit: Can we keep the imports sorted? :)

If we can also piggyback swapping these two lower down in the file, I would love it:
```python
from kedro.io import DataCatalog
from kedro.extras.datasets.spark import SparkDataSet
```

Reply:
@deepyaman done.